### PR TITLE
Add text normalizer transformer to AutoML

### PR DESCRIPTION
**Pull request by zewditu**
We are excited to review your PR.

So we can do the best job, please check:

- [ ] There's a descriptive title that will make sense to other developers some time from now. 
- [ ] There's associated issues. All PR's should have issue(s) associated - unless a trivial self-evident change such as fixing a typo. You can use the format `Fixes #nnnn` in your description to cause GitHub to automatically close the issue(s) when your PR is merged.
- [ ] Your change description explains what the change does, why you chose your approach, and anything else that reviewers should know.
- [ ] You have included any necessary tests in the same PR.

### CODE REVIEW

File: tools-local/Microsoft.ML.AutoML.SourceGenerator/SearchSpaceGenerator.cs [hunk 1]
```diff
@@ -78,6 +79,8 @@ public void Execute(GeneratorExecutionContext context)
                         (_, "Microsoft.ML.Vision.ImageClassificationTrainer.Architecture") => defaultToken.GetValue<string>(),
                         (_, "Microsoft.ML.Data.DataKind") => defaultToken.GetValue<string>(),
                         (_, "Microsoft.ML.IDataView") => defaultToken.GetValue<string>(),
+                        (_, "Microsoft.ML.Transforms.Text.TextNormalizingEstimator.CaseMode") => defaultToken.GetValue<string>(),
+
```
Comment (LittleLittleCloud) [inline tools-local/Microsoft.ML.AutoML.SourceGenerator/SearchSpaceGenerator.cs:83]:
```suggestion
```

File: src/Microsoft.ML.AutoML/CodeGen/normalize_text_search_space.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      "name": "InputColumnName",
+      "type": "string"
+    },
+    {
+      "name": "OutputColumnName",
+      "type": "string"
+    },
+    {
+      "name": "Mode",
+      "type": "caseMode"
```
Comment (LittleLittleCloud) [inline src/Microsoft.ML.AutoML/CodeGen/normalize_text_search_space.json:15]: Does Mode also has default value?

File: src/Microsoft.ML.AutoML/Microsoft.ML.AutoML.csproj [hunk 3]
```diff
@@ -69,6 +69,10 @@
     <AdditionalFiles Include="CodeGen\code_gen_flag.json" />
     <AdditionalFiles Include="CodeGen\*-estimators.json" />
   </ItemGroup>
+
```
Comment (LittleLittleCloud) [inline src/Microsoft.ML.AutoML/Microsoft.ML.AutoML.csproj:72]: Remove this change please

**Comment by zewditu** [comment]
I have a build error 
image
 Is that because of those extra spaces ?

**Review by LittleLittleCloud — APPROVED** [review]

**Pull request merged**